core/src/main/java/io/jenkins/pluginhealth/scoring/model/updatecenter/SecurityWarning.java

```diff
@@ -29,6 +29,11 @@
 public record SecurityWarning(
     String id,
     String name,
+
+    String url,
     List<SecurityWarningVersion> versions
 ) {
+    public SecurityWarning(String name, String url) {
+        this(null, name, url, List.of());
+    }
```

Comment:
I don't think we want to get rid of the ID, so please don't initialize it empty.
 Also, initializing the `versions` field to an empty list will compromise the code which detects if the current version of the plugin is concerned by the security issue.

```suggestion
```